front/src/services/api/search/search.ts

```diff
@@ -18,18 +18,20 @@ export const SEARCH_PEOPLE_QUERY = gql`
 `;
 
 export const SEARCH_USER_QUERY = gql`
-  query SearchQuery($where: users_bool_exp, $limit: Int) {
+  query SearchUserQuery($where: users_bool_exp, $limit: Int) {
     searchResults: users(where: $where, limit: $limit) {
       id
       email
       displayName
     }
   }
 `;
-
+// TODO: remove this query
 export const EMPTY_QUERY = gql`
```

Comment:
this is a hack we should refactor. If I leave it with _, the graphql codegen is actually refusing to work. Technical debt to be removed here